torch/autograd/gradcheck.py

```diff
@@ -63,9 +63,6 @@ def get_numerical_jacobian(fn, input, target=None, eps=1e-3):
 
     # TODO: compare structure
     for x_tensor, d_tensor in zip(x_tensors, j_tensors):
```

Comment:
gradcheck for complex needs more investigation (removing this would mean running tests with vector and covector 1.0 + 0j)